Dockerfile

```diff
@@ -89,4 +89,4 @@ testnet=${TESTNET}\n" > ${ROOTDATADIR}/.digibyte/digibyte.conf
 #RUN ln -s /usr/local/bin/digibyted /usr/bin/digibyted
 #RUN ln -s /usr/local/bin/digibyte-cli /usr/bin/digibyte-cli
 
-CMD /usr/local/bin/digibyted
+CMD /usr/local/bin/digibyted -conf=${ROOTDATADIR}/.digibyte/digibyte.conf
```

Comment:
@ChillingSilence Shouldn't we specify the config file like so? Or is it automatically using the config from the directory `$ROOTDATADIR `?

Reply:
AFAIK @SmartArray it's technically in the WORKDIR which we have to define as the $ROOTDATADIR ARG, but yeah it can't hurt to launch it as ^^ just to be safe :)

Reply:
Good to know!